src/content/docs/en/reference/experimental-flags/fonts.mdx

```diff
@@ -216,6 +216,65 @@ provider: fontProviders.google({
 
 You can also [make a custom Astro font provider](#build-your-own-font-provider) for any unifont provider.
 
+## Usage examples
+
+```js title="astro.config.mjs"
+import { defineConfig, fontProviders } from "astro/config";
+
+export default defineConfig({
+  experimental: {
+    fonts: [
+      {
+        name: "Roboto",
+        cssVariable: "--font-roboto"
+        provider: fontProviders.google(),
```

Comment:
```suggestion
        provider: fontProviders.google(),
        // Default included:
        // weights: [400] ,
        // styles: ["normal", "italics"],
        // subsets: ["cyrillic-ext", "cyrillic", "greek-ext", "greek", "vietnamese", "latin-ext", "latin"],
        // fallbacks: ["sans-serif"], 
```
When we discussed having a "minimal" example, we talked about showing the defaults. Do you think this is helpful?